src/tests/utils/ApiTestSteps.ts

```diff
@@ -319,14 +319,17 @@ export async function validateTxMAEventData(keyList: any, yotiMockID: any): Prom
 	}
 }
 
-export async function validateTxMAEvent(txmaEvent: any, keyList: any, yotiMockId: any): Promise<any> {
+export async function validateTxMAEvent(txmaEvent: any, keyList: any, yotiMockId: any, failedCheck: Boolean, vcData: any): Promise<any> {
```

Comment:
think this need to be a lower case b `boolean`

Reply:
Also would we be able to update the other types (as much as possible)? `txmaEvent`and `yotiMockId` looks like strings, not sure about the others

Reply:
Updated all known types based on suggestions